### DISCUSSION TITLE
youtube bb google drive

### DISCUSSION THREAD

**Discussion by JayJayBinks1** [STVIR/pysot#426] [General]
Is there a Google Drive link for the youtube bb dataset?

If not, what is the link I should use to download youtube_bb over ssh? I can't read Chinese so I'm struggling to figure out how to do this.

**Comment by ZhiyuanChen** [comment]
Ahhhhhh, sorry about that, I will need to talk to other maintainers about this

**Reply by JayJayBinks1** [reply]
I was able to get a Chinese friend to download the dataset and send it to me. It would be helpful if it was on Google Drive though.

Can I ask what format the data is expected to be in? Can I simply just unzip the tar files and run it?

**Reply by ZhiyuanChen** [reply]
Sorry for this late reply. All you need to do is just unzip it

**Comment by IlchaeJung** [comment]
I am very appreciate at your valuable works.
Unfortunately, there is hardness to access baidu cloud from outside of China. Therefore, our model cannot reach the original performance...TT. Could you please upload the dataset on google drive? Or.. anybody who already downloaded the dataset can share your files?

Best regards,
Ilchae Jung
